users/reallyinvincible/Task_3/.idea/Task_3.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
```

Comment:
Why all these are necessary? I think you are using an IDE which is making the `.idea` files, use `.gitignore` or delete them before committing

Reply:
Yup, WebStorm. Will take care next time.

Reply:
Deleted.